src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -17,31 +21,50 @@
 public class Intake extends SubsystemBase {
 
   private static Intake m_intake;
-  private CANSparkMax m_leftIntakeRollerMotor;
-  private CANSparkMax m_rightIntakeRollerMotor;
-  public static final double DEFAULT_ROLLER_SPEED = 0.5;
+  private CANSparkMax m_intakeRollerMotor;
+  private DoubleSolenoid m_piston1;
+  private DoubleSolenoid m_piston2;
+  public static final double DEFAULT_ROLLER_SPEED = 0.3;
+  private boolean intakeFuture = false;
   
   /** Creates a new Intake. */
   private Intake() {
-    m_leftIntakeRollerMotor = new CANSparkMax(RobotMap.I_LEFT_ROLLER, MotorType.kBrushless);
-    m_rightIntakeRollerMotor = new CANSparkMax(RobotMap.I_RIGHT_ROLLER, MotorType.kBrushless);
+    m_intakeRollerMotor = new CANSparkMax(RobotMap.I_LEFT_ROLLER, MotorType.kBrushless);
+    if(intakeFuture == true){
+    m_piston1 = new DoubleSolenoid(RobotMap.B_PCM_CAN, PneumaticsModuleType.CTREPCM, RobotMap.I_PISTON_PORT1, RobotMap.I_PISTON_PORT2);
+    m_piston2 = new DoubleSolenoid(RobotMap.B_PCM_CAN, PneumaticsModuleType.CTREPCM, RobotMap.I_PISTON_PORT1, RobotMap.I_PISTON_PORT2);
```

Comment:
Let's call these m_leftPiston and m_rightPiston . Also, the I_PISTON_PORT1 and the I_PISTON_PORT2 need to be different between these two pistons as well.